src/Plugin.Permissions.iOSUnified/PermissionsImplementation.cs

```diff
@@ -137,6 +139,9 @@ public Task<PermissionStatus> CheckPermissionStatusAsync(Permission permission)
                     case Permission.Sensors:
                         results.Add(permission, await RequestSensorsPermission().ConfigureAwait(false));
                         break;
+                    case Permission.Speech:
+                        results.Add(permission, await this.RequestSensorsPermission().ConfigureAwait(false));
```

Comment:
Correct - copy/paste job.  Should have been 

results.Add(permission, await this.RequestSpeechPermission().ConfigureAwait(false));

I can submit another PR or it can be resolved post merge.  Your call.